### ISSUE TITLE
Reliable feature check detection

### ISSUE THREAD

**Issue by jclem** [jclem/detour#16]
Currently, [`ActiveRecord::Rollout::Feature.all-with_lines`](https://github.com/jclem/active_record_rollout/blob/9b53053a59057abc19d64a45cc14db6d547fe55c/lib/active_record/rollout/feature.rb#L135) can only detect feature checks of these forms:

``` ruby
foo.has_feature? :foo
foo.has_feature?(:foo)
```

Meaning that forms such as these will not register (with our without parentheses):

``` ruby
foo.has_feature? "foo"
foo.has_feature? "Feature Name"
```

Not sure if this is a bug, or if it should just validate that feature names be composed of alphanumeric characters and underscores. It's easy enough to at least update the regex to also find:

``` ruby
foo.has_feature? "foo_bar"
```

**Comment by jclem** [comment]
I'm fine with only looking for symbols.

**Comment by jclem** [comment]
Nope, reopening.

**Comment by jclem** [comment]
Closed by e11ddad
